docker-compose.yml

```diff
@@ -26,15 +26,37 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_DB: datastore_local
 
-  runner-app:
-    container_name: runner-app
+  forms-email-app:
+    container_name: forms-email-app
     build:
-      context: .runner
+      context: .forms/email-output
       dockerfile: Dockerfile
     ports:
-      - 3000:3000
+      - 5080:3000
```

Comment:
Putting these onto a very different port (which will not collide with `fb-component-acceptance-tests` when they are combined)